package.json

```diff
@@ -48,6 +48,7 @@
     "@uiw/react-markdown-preview": "^4.1.13",
     "@uiw/react-md-editor": "^3.14.3",
     "@venusprotocol/isolated-pools": "^1.1.0",
+    "@venusprotocol/oracle": "^1.7.0-dev.5",
```

Comment:
New main release will be published soon with the needed changes. See https://github.com/VenusProtocol/oracle/pull/112

Reply:
Ok great, thanks for the heads up

Reply:
oracles 1.7.0 is ready: https://github.com/VenusProtocol/oracle/releases/tag/v1.7.0

Reply:
Nice, thanks